src/main/rx/srxl2.c

```diff
@@ -210,6 +209,7 @@ bool srxl2ProcessControlData(const Srxl2Header* header, rxRuntimeState_t *rxRunt
 
     case FailsafeChannelData: {
         srxl2ProcessChannelData((const Srxl2ChannelDataHeader *) (controlData + 1), rxRuntimeState);
+        globalResult |= RX_FRAME_FAILSAFE;
```

Comment:
Just a quick question: Why is `srxl2ProcessChannelData()` called above here? If the link is in a failsafe condition, there should be no new RC data, and Betaflight won't be using it anyway while in failsafe mode.

Reply:
It does not necessarily have to be called. Just makes the failsafe channel data available in case betaflight want's to use it. I have no problems removing it if betaflight has no future intention of using the receivers own failsafe channel values for any purpose.

Reply:
In terms of receiver behavior, the default is that throttle goes to a preset position, usually a low/idle value (whatever it was set to on the transmitter when it was bound to the receiver), and everything else stays at the last received position.  I say default, as we may give users the option to modify this for serial receivers in the future.

Reply:
The expected RX behaviour in Betaflight is that in case of a failsafe the RX stops sending RC channel values, and does not repeat the last good values.
In cases like SRXL2 where the RX explicitly sets `RX_FRAME_FAILSAFE` this isn't strictly required, as channel values are ignored when this flag is set. But I think it will make the code easier to understand if the `srxl2ProcessChannelData()` inside `FailsafeChannelData` is removed.